### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by frankie303**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -2902,7 +2902,7 @@ export class ZodEffects<
   constructor(def: ZodEffectsDef<T>) {
     super(def);
     // if (def.schema instanceof ZodEffects) {
-    //   throw new Error(ZodFirstPartyTypeKind.ZodEffectscannot be nested.");
+    //   throw new Error(ZodFirstPartyTypeKind.ZodEffects cannot be nested.");
```
Comment (scotttrinh) [inline src/types.ts:2905]: Can you revert this one? It's actually invalid even with the typo fixed, so no reason to add more git history to this line.

**Review by scotttrinh — CHANGES_REQUESTED** [review]
Thanks for taking the time to PR this! One small request.

File: src/types.ts (hunk 1 above)
Comment (frankie303) [reply src/types.ts:2905]: Sure I can revert it but just curious, what is the side effect of adding one more line to git history in this case?

Comment (scotttrinh) [reply src/types.ts:2905]: Not a huge deal, just makes doing a `git blame` on this chunk of commented out code have a random typo fix commit in the middle of it. Definitely not an actual problem 😅

Comment (colinhacks) [reply src/types.ts:2905]: Oops sorry Scott, saw this review immediately after merging. This whole comment is getting deleted in v3.9 so doesn't really matter.

**Pull request merged**